### PR TITLE
docs: updated change log for levski release

### PR DESCRIPTION
**Pull request by brianointel**
Signed-off-by: brianointel <brian.j.osburn@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by bill-mahoney — DISMISSED** [review]
LGTM

**Review by presatish — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,46 @@
...
+- **snap:** add config interface with unique identifier ([#103](https://github.com/edgexfoundry/device-usb-camera/issues/103)) ([#b346198](https://github.com/edgexfoundry/device-usb-camera/commits/b346198))
+
+### Code Refactoring ♻
+- **snap:** Simplify rtsp server versioning and configuration ([#1dbc6f5](https://github.com/edgexfoundry/device-usb-camera/commits/1dbc6f5))
+
+### Bug Fixes 🐛
+- improve error messaging on read and write commands ([#116](https://github.com/edgexfoundry/device-usb-camera/issues/116)) ([#a13c0d8](https://github.com/edgexfoundry/device-usb-camera/commits/a13c0d8))
+- address pr comments and add timer ([#81c0ea8](https://github.com/edgexfoundry/device-usb-camera/commits/81c0ea8))
+- return if stop error ([#679fd9a](https://github.com/edgexfoundry/device-usb-camera/commits/679fd9a))
+- startStreaming now returns all errors ([#50aed43](https://github.com/edgexfoundry/device-usb-camera/commits/50aed43))
```
Comment (ChristianDarr-personal) [inline CHANGELOG.md]:
```suggestion
- error forwarding on startStreaming command ([#113] https://github.com/edgexfoundry/device-usb-camera/issues/113) ([#02bc335] (https://github.com/edgexfoundry/device-usb-camera/commit/02bc3351eb583ffe88737b5638435757cc287900)
```

Comment (ChristianDarr-personal) [reply CHANGELOG.md]: So I just noticed this, but it looks like my fix for issue #23 did not get squashed and merged, so all 3 of my commits are sitting in the repo and are not attributed to one single pr (#113). Is this an issue?

**Review by ChristianDarr-personal — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (presatish) [reply CHANGELOG.md]: I think whatever you have suggested is good, not seeing any issue.

**Review by ChristianDarr-personal — APPROVED** [review]

**Comment by lenny-goodell** [comment]
recheck

**Pull request merged**